apps/estuary/lib/estuary/application.ex

```diff
@@ -2,70 +2,12 @@ defmodule Estuary.Application do
   @moduledoc false
   use Application
 
-  alias Estuary.EventTable
-
-  @spec start(any, any) :: {:error, any} | {:ok, pid}
   def start(_type, _args) do
-    if elsa_endpoint() != nil do
-      validate_topic_exists()
-      EventTable.create_schema()
-      EventTable.create_table()
-    end
-
-    children = get_children()
-
-    opts = [strategy: :one_for_one, name: Estuary.Supervisor]
-    Supervisor.start_link(children, opts)
-  end
-
-  defp elsa_options do
     [
-      endpoints: elsa_endpoint(),
-      connection: :estuary_elsa,
-      producer: [topic: event_stream_topic()],
-      group_consumer: [
-        group: "estuary-consumer-group",
-        topics: [event_stream_topic()],
-        handler: Estuary.MessageHandler,
-        config: [
-          begin_offset: :earliest,
-          offset_reset_policy: :reset_to_earliest
-        ]
-      ]
+      {DynamicSupervisor, strategy: :one_for_one, name: Estuary.Dynamic.Supervisor},
```

Comment:
i don't think i see anything using this dynamic supervisor. why is it here in your supervision tree?

Reply:
We removed it.